civihr_default_theme/template.php

```diff
@@ -956,3 +956,19 @@ function _set_maximum_scale_to_viewport_meta_tag(&$head_elements) {
   // Set updated rules to the tag
   $viewportTagValue = implode(', ', $rules);
 }
+
+/**
+ * Implements hook_form_alter()
+ *
+ * @param array $form
+ * @param array $form_state
+ * @param string $form_id
+ */
+function civihr_default_theme_form_alter(&$form, $form_state, $form_id) {
```

Comment:
@deb1990 move this to employee portal module ( inside .module file check if its not added already  ) please.

Reply:
Done